authoidc.properties

```diff
@@ -1,12 +1,18 @@
 category=Integration
 description=Delegate authentication using the OpenID Connect protocol
 homepageUrl=https://github.com/vaulttec/sonar-auth-oidc
-archivedVersions=1.0.4,1.1.0,2.0.0
-publicVersions=2.1.0
+archivedVersions=1.0.4,1.1.0,2.0.0,2.1.0
+publicVersions=2.1.1
 
 defaults.mavenGroupId=org.vaulttec.sonarqube.auth.oidc
 defaults.mavenArtifactId=sonar-auth-oidc-plugin
 
+2.1.1.description=Fixes some issues with the newly added auto-login servlet filter
+2.1.1.sqVersions=[7.9,LATEST]
+2.1.1.date=2021-10-24
```

Comment:
This date doesn't look right to me. You posted the request in the community on 28 December.

Reply:
Yes, it took me 2 month (after the release of this plugin version on 2021-10-24) to prepare the steps for the marketplace entry :)
<img width="1274" alt="Screenshot 2022-01-06 at 23 15 19" src="https://user-images.githubusercontent.com/2626998/148460138-d04ae4d0-4d38-4d2a-a6c3-0375b76baf1c.png">

Reply:
This date should be (approximately) when the plugin became available in the Marketplace.

Reply:
Ok, I'll fix this...